errai-jaxrs/errai-jaxrs-client/pom.xml

```diff
@@ -61,8 +64,8 @@
               <artifactItems>
                 <artifactItem>
                   <groupId>org.jboss.spec.javax.ws.rs</groupId>
-                  <artifactId>jboss-jaxrs-api_1.1_spec</artifactId>
-                  <version>1.0.1.Final</version>
+                  <artifactId>jboss-jaxrs-api_2.0_spec</artifactId>
+                  <version>1.0.0.Final</version>
                   <classifier>sources</classifier>
                   <overWrite>false</overWrite>
                   <outputDirectory>target/generated-sources/javax/ws/rs/super</outputDirectory>
```

Comment:
This changes were necessary to run the integration tests inside IntelliJ. For the record, I also had to manually copy the `war/WEB-INF/web.xml` file to the directory IntelliJ uses as output directory for tests and mark the `src/main/java/org/jboss/errai/enterprise/compat` as an exclusion. The JVM parameter `-Derrai.marshalling.use_native_javac=true` was also necessary for me.